### PR TITLE
Fix python module building

### PR DESCRIPTION
**Pull request by hjmjohnson**

### CODE REVIEW

**Comment by hjmjohnson** [comment]
Verified that this fixes many of the common test failures on a local machine.

**Comment by hjmjohnson** [comment]
```
+ rm dist/itk_polartransform-1.1.1-cp37-cp37m-linux_x86_64.whl
+ for whl in 'dist/*linux_$(uname -p).whl'
+ auditwheel repair dist/itk_polartransform-1.1.1-cp38-cp38-linux_x86_64.whl -w /work/dist/
INFO:auditwheel.main_repair:Repairing itk_polartransform-1.1.1-cp38-cp38-linux_x86_64.whl
INFO:auditwheel.wheeltools:Previous filename tags: linux_x86_64
INFO:auditwheel.wheeltools:New filename tags: manylinux1_x86_64
INFO:auditwheel.wheeltools:Previous WHEEL info tags: cp38-cp38-linux_x86_64
INFO:auditwheel.wheeltools:New WHEEL info tags: cp38-cp38-manylinux1_x86_64
INFO:auditwheel.main_repair:
Fixed-up wheel written to /work/dist/itk_polartransform-1.1.1-cp38-cp38-manylinux1_x86_64.whl
+ rm dist/itk_polartransform-1.1.1-cp38-cp38-linux_x86_64.whl
```

**Comment by hjmjohnson** [comment]
@thewtex @dzenanz This is a bottleneck to testing remote modules python generation.  None of the remote modules can pass CI testing until these scripts are made to work again.

File: scripts/dockcross-manylinux-download-cache-and-build-module-wheels.sh.orig [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  curl https://data.kitware.com/api/v1/file/592dd8068d777f16d01e1a92/download -o zstd-1.2.0-linux.tar.gz
+  gunzip -d zstd-1.2.0-linux.tar.gz
+  tar xf zstd-1.2.0-linux.tar
+fi
+if [[ ! -f ./zstd-1.2.0-linux/bin/unzstd ]]; then
+  echo "ERROR: can not find required binary './zstd-1.2.0-linux/bin/unzstd'"
+  exit 255
+fi
+
+<<<<<<< HEAD
```
Comment (jhlegarreta) [inline scripts/dockcross-manylinux-download-cache-and-build-module-wheels.sh.orig]: @hjmjohnson this seems to be an oversight from a rebase/merge.

File: scripts/dockcross-manylinux-download-cache-and-build-module-wheels.sh.orig [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+fi
+if [[ ! -f ./ITKPythonBuilds-linux.tar.zst ]]; then
+  echo "ERROR: can not find required binary './ITKPythonBuilds-linux.tar.zst'"
+  exit 255
+fi
+if [[ ! -f ./ITKPythonPackage/scripts/dockcross-manylinux-build-module-wheels.sh ]]; then
+  echo "ERROR: can not find required binary './ITKPythonPackage/scripts/dockcross-manylinux-build-module-wheels.sh'"
+  exit 255
+fi
+=======
```
Comment (jhlegarreta) [inline scripts/dockcross-manylinux-download-cache-and-build-module-wheels.sh.orig]: Rebase/merge trace.

File: scripts/dockcross-manylinux-download-cache-and-build-module-wheels.sh.orig [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+curl -L https://github.com/InsightSoftwareConsortium/ITKPythonBuilds/releases/download/${ITK_PACKAGE_VERSION:=v5.1rc02}/ITKPythonBuilds-linux.tar.zst -O
+./zstd-1.2.0-linux/bin/unzstd ITKPythonBuilds-linux.tar.zst -o ITKPythonBuilds-linux.tar
+tar xf ITKPythonBuilds-linux.tar
+
+mkdir tools
+curl https://data.kitware.com/api/v1/file/5c0aa4b18d777f2179dd0a71/download -o doxygen-1.8.11.linux.bin.tar.gz
+tar -xvzf doxygen-1.8.11.linux.bin.tar.gz -C tools
+>>>>>>> 481e86e... ENH: Choose v5.1rc02 python package
```
Comment (jhlegarreta) [inline scripts/dockcross-manylinux-download-cache-and-build-module-wheels.sh.orig]: Rebase/merge trace.

File: scripts/dockcross-manylinux-download-cache-and-build-module-wheels.sh.orig (hunk 1 above)
Comment (jhlegarreta) [reply scripts/dockcross-manylinux-download-cache-and-build-module-wheels.sh.orig]: Fixed in 940e651.

File: scripts/dockcross-manylinux-download-cache-and-build-module-wheels.sh.orig (hunk 2 above)
Comment (jhlegarreta) [reply scripts/dockcross-manylinux-download-cache-and-build-module-wheels.sh.orig]: Fixed in 940e651.

File: scripts/dockcross-manylinux-download-cache-and-build-module-wheels.sh.orig (hunk 3 above)
Comment (jhlegarreta) [reply scripts/dockcross-manylinux-download-cache-and-build-module-wheels.sh.orig]: Fixed in 940e651.

**Pull request merged**